### PR TITLE
IE fix, reorganize package.json

### PR DESCRIPTION
**Pull request by devinleighsmith**
missed committing this on Monday, but would rather commit this separately to keep the review small.

### CODE REVIEW

File: frontend/public/keycloak.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-  "realm": "xz0xtue5",
-  "auth-server-url": "https://sso-dev.pathfinder.gov.bc.ca/auth",
+  "realm": "quartech",
```
Comment (Fosol) [inline frontend/public/keycloak.json]: I don't think this realm is correct for OpenShift

File: frontend/public/keycloak.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+  "auth-server-url": "http://localhost:8080/auth",
```
Comment (Fosol) [inline frontend/public/keycloak.json]: I don't think this URL is correct for OpenShift

File: frontend/public/keycloak.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
   "ssl-required": "external",
-  "resource": "pims-app",
+  "resource": "team-blue-app",
```
Comment (Fosol) [inline frontend/public/keycloak.json]: I don't think this resource is correct for OpenShift

File: frontend/public/keycloak.json (hunk 1 above)
Comment (devinleighsmith) [reply frontend/public/keycloak.json]: This file is now only used locally, so I switched the config back to the local keycloak config.

**Pull request merged**